packages/js/data/src/plugins/actions.ts

```diff
@@ -416,8 +417,8 @@ export type Actions =
 	| ReturnType< typeof setRecommendedPlugins >;
 
 // Types
-export type ActionDispatchers = {
+export type ActionDispatchers = DispatchFromMap< {
```

Comment:
`DispatchFromMap` can map a "raw" actionCreators object to the actions available when registered on the @wordpress/data store.